### ISSUE TITLE
Netboot Raspberry pi?!

### ISSUE THREAD

**Issue by Geethree** [Flux-Foundation/Dreamland-Biennale#4]
Very interesting. Would be curious to spend a few cycles to get this to work.. https://github.com/raspberrypi/documentation/blob/master/hardware/raspberrypi/bootmodes/net.md

**Comment by Geethree** [comment]
@fragmede Is this why you were picking up an RPi 3?

**Comment by fragmede** [comment]
Yeah, I have a theory or two on how to do the configuration for netboot to
associate a given pi3, its serial number, and the fixture it's located at,
but it's going to need some configuration and testing.

https://amazon.com/Ubiquiti-EdgeSwitch-ES-24-250W-24-Ports-Managed/dp/B00LV8Z2V2
is $320 if we wanted to use vlans to tag it.

Another possibility is only use netboot to setup new pi's. Like, still use
SD cards, as that's easy to understand - "this SD card does $thing", but
for new unknown pi's, netboot them. If there's exactly 1 pi missing from
the system, and we see a new pi getting added, then we could make an
educated guess about what's going on.

**Comment by Geethree** [comment]
We could grab the CPU's serial from /proc/cpuinfo perhaps?
